### PR TITLE
Apply black styling to test_N*.py

### PR DESCRIPTION
**Pull request by jvfe**
This pull request addresses issue #2552 and #2835 

- [X] I hereby agree to dual licence this and any previous contributions under both
the _Biopython License Agreement_ **AND** the _BSD 3-Clause License_.

- [X] I have read the ``CONTRIBUTING.rst`` file, have run ``flake8`` locally, and
understand that AppVeyor and TravisCI will be used to confirm the Biopython unit
tests and style checks pass with these changes.

- [X] I have added my name to the alphabetical contributors listings in the files
``NEWS.rst`` and ``CONTRIB.rst`` as part of this pull request, am listed
already, or do not wish to be listed. (*This acknowledgement is optional.*)

### CODE REVIEW

**Comment by peterjc** [comment]
flake8 is unhappy, quoting the TravisCI style check output:

```
$ flake8
./Tests/test_Nexus.py:724:48: W291 trailing whitespace
./Tests/test_Nexus.py:725:48: W291 trailing whitespace
./Tests/test_Nexus.py:726:48: W291 trailing whitespace
./Tests/test_Nexus.py:727:48: W291 trailing whitespace
./Tests/test_Nexus.py:728:46: W291 trailing whitespace
```

File: Tests/test_Nexus.py [hunk 1]
```diff
@@ -371,11 +727,17 @@ def test_NexusTest1(self):
 [T]    2.43     2.12     2.31      .       2.14     
 [-]    2.43     2.14     2.31     2.14      .       
 ;
-""")  # noqa : W291
+""",
+        )  # noqa : W291
```
Comment (peterjc) [inline Tests/test_Nexus.py:731]: You will need to manually move the comment ``# noqa : W291`` up one line to be where the multi-line string ends

**Comment by peterjc** [comment]
I see why now (we've had this before with black). You'll have to manually move five special comments ``# noqa : W291`` up one line to be where the multi-line string ends.

File: Tests/test_Nexus.py [hunk 2]
```diff
@@ -647,7 +1136,8 @@ def test_repeated_names_no_taxa(self):
         CYS1_DICDI          -----MKVIL LFVLAVFTVF VSS------- --------RG IPPEEQ---X
         ;
         end;
-        """)  # noqa : W291
+        """
+        )  # noqa : W291
```
Comment (peterjc) [inline Tests/test_Nexus.py:1140]: Strange. Looks like we can remove this ``# noqa : W291`` and the others still on a closing bracket line?

Comment (jvfe) [reply Tests/test_Nexus.py:1140]: Looks like the only one that's needed is on L730, removed the other two (L1140 and L1170) and flake8 didn't complain. I'll make another commit removing them.

**Pull request merged**